build/create_binaries/add_metadata.jl

```diff
@@ -72,4 +76,23 @@ function add_metadata(project_dir, license_file, output_dir, git_repo)
         ```"""
         println(io, version_info)
     end
+
+    # collect lisences of all dependencies
+    ctx = PackageCompiler.create_pkg_context(project_dir)
+    license_dir = joinpath(output_dir, "dep_licenses") 
+    mkpath(license_dir)
+    for (uuid,pkg_entry) in ctx.env.manifest.deps
+	if isnothing(pkg_entry.tree_hash)
+	    # seems as though stdlib packages don't have a tree_sha. as there doesn't seem to be 
+	    # a different way of detecting those, I'm going to assume that is characteristic 
```

Comment:
Related to this, there is this file https://github.com/JuliaLang/julia/blob/master/THIRDPARTY.md

And I think @deltamarnix opened an issue at PkgToSoftwareBOM.jl to see if it could be modified to also cover these libraries, but I don't think there has been any progress